src/modules/welcome/Config.cpp

```diff
@@ -27,16 +27,19 @@
 void
 RequirementsModel::setRequirementsList( const Calamares::RequirementsList& requirements )
 {
+    CALAMARES_RETRANSLATE_SLOT( &RequirementsModel::retranslate )
```

Comment:
This should only be called from a UI method, and generally only once (e.g. when making the UI). If the model needs to emit changed (translated) messages then of course it can (e.g. `data()` can return translated strings, and probably **should** do so for display-role). The re-translating slot should force a re-draw of the UI or a re-query of data.

Reply:
got it